src/frontend/js/revisions.ts

```diff
@@ -56,7 +56,7 @@ function handleRevisionSliderInput({ target }: Event) {
     .getAttribute("data-date");
   // Update the revision info box
   revisionInfo.textContent =
-  `Revision: ${currentRevision}\r\n${revisionDate}`;
+  `Verision: ${currentRevision}\r\n${revisionDate}`;
```

Comment:
```suggestion
  document.getElementById("revision-number").textContent = currentRevision.toString();
  document.getElementById("revision-editor").textContent = revisionElement.getAttribute("data-editor");
  document.getElementById("revision-date").textContent = revisionElement.getAttribute("data-date");
```
Since I splitted the info fields in separate spans in the `<output>`-field, they have to be set separately now.
Also, please change the lines above
```
  // The last updated date of the revision
  const revisionDate = document
    .getElementById("revision-" + currentRevision)
    .getAttribute("data-date");
```
to
```
  // The revision element
  const revisionElement = document.getElementById("revision-" + currentRevision)
```
(I cannot make an automated suggestion here because it's too many lines away from your changes.)